klippa/src/lib.rs

```diff
@@ -157,10 +158,12 @@ impl Plan {
         input_unicodes: &IntSet<u32>,
         font: &FontRef,
         flags: SubsetFlags,
+        drop_tables: &IntSet<u32>,
```

Comment:
we should either make `Tag` work with `IntSet`, or just use a HashMap? I'd be very surprised if this showed up on a profile..

Reply:
I think for non-performance critical parts BTreeSet<Tag> is slightly preferred as int set's + tags can consume excessive memory due to needing approximately one 512 bit page per tag. Using a btree will keep tag order sorted which is generally what we want.

Reply:
Looking at the usage here we don't depend on ordering anywhere so HashSet<Tag> would also be totally fine.

Reply:
I used IntSet for convenience and would happily change it to HashSet. Though I have one question: what is the best way to use HashSet when we have `--drop-tables=*, --drop-tables-=some_table`

Reply:
HashSet doesn't support `all()`

Reply:
Thanks! I'll work on this

Reply:
Done, can you help review again?